gql_code_builder/lib/src/operation/data.dart

```diff
@@ -358,15 +378,18 @@ Method _buildGetter(
     final unwrappedTypeNode = _unwrapTypeNode(typeNode);
     final typeName = unwrappedTypeNode.name.value;
     final fieldTypeDef = _getTypeDefinitionNode(
-      schema,
+      schemaSource.document,
       typeName,
     );
     final typeMap = {
       ...defaultTypeMap,
       if (node.selectionSet != null)
         typeName: refer("$prefix\$$name")
       else if (fieldTypeDef != null)
-        typeName: refer(typeName, schemaUrl)
+        typeName: refer(
+          typeName,
+          schemaSource.url.replaceAll(RegExp(r".graphql$"), ".schema.gql.dart"),
```

Comment:
Same thing here - how do you suggest we pass the extension?

Reply:
Just add a code comment for now.